code/modules/food_and_drink/cakes.dm

```diff
@@ -589,7 +589,7 @@
 
 /obj/item/reagent_containers/food/snacks/cake/chocolate/gateau
 	name = "Extravagant Chocolate Gateau"
-	desc = "Holy shit! This cake probably costs more than the gross domestic product of Bulgaria!"
+	desc = "Holy shit! This cake probably costs more than the peak market cap of the Syndicate!"
```

Comment:
"The Syndicate" is a fictional terrorist/paramilitary/(etc.) organization bankrolled by corporations with a bone to pick with NT, not a corporation itself.

Reply:
https://wiki.ss13.co/Corporations 
@klushy225 maybe this is useful? Donk Co. instead? other crimiminal companies?

Reply:
Updated to Donk Co. Other alternatives could be: "... probably costs more than this station!", "... probably costs more than your organs!". Etc